src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -47,7 +47,9 @@ public abstract partial class QuicConnectionOptions
         public System.TimeSpan KeepAliveInterval { get { throw null; } set { } }
         public int MaxInboundBidirectionalStreams { get { throw null; } set { } }
         public int MaxInboundUnidirectionalStreams { get { throw null; } set { } }
+        public System.Net.Quic.QuicConnectionStreamsAvailableCallback? StreamsAvailableCallback { get { throw null; } set { } }
     }
+    public delegate void QuicConnectionStreamsAvailableCallback(System.Net.Quic.QuicConnection sender, int bidirectionalStreamsCountIncrement, int unidirectionalStreamsCountIncrement);
```

Comment:
```suggestion
    public delegate void QuicConnectionStreamsAvailableCallback(System.Net.Quic.QuicConnection connection, int bidirectionalStreamsCountIncrement, int unidirectionalStreamsCountIncrement);
```

Nit: I don't think we use "sender" much outside of event handlers using `object?`.

Reply:
Done